docs/running-node-ansible.md

```diff
@@ -0,0 +1,204 @@
+# Running a DCLedger Node using Ansible
+
+This document describes in how to:
+
+* configure different types of DCLedger nodes: genesis, validator and observer
+* create a node administrator (or admin) account as a necessary part of validator and genesis node configuration
+
+## Requirements (local machine)
+
+### 1. Clone [DCL][5] project
+```
+git clone https://github.com/zigbee-alliance/distributed-compliance-ledger.git
+```
+
+### 2. Install `Python` and `Pip`
+```bash
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends python3
+sudo apt install python3-pip
+sudo apt install python3-testresources
```

Comment:
Any reason to install `testresources` package from the OS distribution instead. There are three alternatives here:
1. Use `pip` to install it:
```suggestion
sudo apt install python3-pip
pip install testresources
```
2. Simplify the `apt` number of instructions
```suggestion
sudo apt install python3-pip python3-testresources
```
3. Include it into the `requirements.in` file